src/test/groovy/lang/ScriptSourcePositionInAstTest.groovy

```diff
@@ -45,18 +46,24 @@ class ScriptSourcePositionInAstTest extends GroovyTestCase {
     }
 
     void testDoubleStatementScript() {
-        assert positionsForScript("""\
+        def raw = """\
             println 'hello'
             println 'bye'
-        """.stripIndent()) == [[1, 1], [2, 14]]
+        """
+        // avoid stripIndent issues on JDK13+ by calling Groovy's DGM stripIndent explicitly
+        def script = StringGroovyMethods.stripIndent((CharSequence)raw)
```

Comment:
I wonder if we should even support JDK13 here. It is not a LTS version, it will stop receiving updates after March 2020 and the stripIdent method in JDK13 is deprecated. I would prefer not changing the tests for that

Reply:
The @Deprecated on `stripIndent` in JDK13 is "just" how they mark preview features. The deprecation appears gone in 14 as the feature has now been more or less accepted. They still have a bit of a disclaimer in the javadoc text itself but I'd assume it is here bar a few tweaks for the foreseeable future. Unless someone else knows different.

Reply:
if it is going to stay then we need to think about how the two stripidents differ and what we do about it. We may have to remove our stripident in general for example

Reply:
In Groovy 3, we added a `stripIndent(boolean)` method (GROOVY-9423). It doesn't make the problem go away but at least provides a workaround. It isn't part of GROOVY_2_5_X hence the need for the slightly hacky change here.